<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="drawPolygon">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Polygon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restorePolygon">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore Polygon</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAll">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erasePolygon">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Polygon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawWindow">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Clipping Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseWindow">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Clipping Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipPolygonSutherHodge">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>380</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip (Sutherland Hodgeman)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipPolygonWeilerAther">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Polygon (Weiler Atherton)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
